profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -4,6 +4,13 @@
  * Webservice definition and behaviour.
  */
 
+// Setting up templates directory path.
+define('TMGMT_POETRY_XML_TEMPLATES_PATH', drupal_get_path('module', 'tmgmt_poetry_mock') . '/xml_templates');
```

Comment:
`tmgmt_poetry` module does not depend from `tmgmt_poetry_mock` so there is something wrong with this constant definition.